INSTALL.rst

```diff
@@ -81,14 +81,18 @@ Using Anaconda
 
     $ conda config --prepend channels conda-forge
 
-7. Create a new conda enviroment.
+7. Set conda channel priority to strict [3]_::
```

Comment:
Can we maybe combine this with the previous step?
```
6. Configure conda to install :mod:`message_ix` from the conda-forge channel [2]_ [3]_::

    $ conda config --prepend channels conda-forge
    $ conda config --set channel_priority strict

…

.. [3] "strict" channel priority ensures that conda will select the latest version of message_ix from the "noarch" channel, and ignore much older (v1.0.0) packages in OS-specific channels.
   See `managing channels`_ of the Anaconda documentation for more information.
```

By explaining the idea of "channel priority" in a footnote only, users don't need to get hung up on understanding what it means. They can simply enter the 2 commands together and get the correct behaviour.